qa/tasks/cephfs/mount.py

```diff
@@ -395,7 +395,14 @@ def cleanup(self):
             else:
                 raise
 
-        self.cleanup_netns()
+        mount_type = str(type(self))
+        # namespaces are optional *only* for vstart clusters.
+        if ('LocalKernelMount' in mount_type or \
+            'LocalFuseMount' in mount_type) and \
+            not self.using_namespace:
+                pass
+        else:
+            self.cleanup_netns()
```

Comment:
Why not overload `cleanup_netns` in `LocalKernelMount`/`LocalFuseMount` to do the check there. We should never be special casing vstart_runner in `qa/tasks/cephfs`

Reply:
Yes, that would be better. Incorporated. Thanks!